website/client/src/components/snackbars/notifications.vue

```diff
@@ -355,10 +356,31 @@ export default {
     }, 16),
 
     updateBannerHeightAndScrollY () {
-      this.gemsPromoBannerHeight = getBannerHeight('gems-promo');
+      this.updateEventBannerHeight();
       this.sleepingBannerHeight = getBannerHeight('damage-paused');
       this.updateScrollY();
     },
+
+    updateEventBannerHeight () {
+      if (this.isEventActive) {
+        this.eventPromoBannerHeight = getBannerHeight(this.currentEventBannerName());
+      }
+    },
+
+    currentEventBannerName () {
+      // if there are any other types of promo bars
```

Comment:
@SabreCat is this here a good place or should this be moved to the events.js ? Then a second export would be to be changed if an event has a promo banner

Reply:
Hmm, I feel like the best thing would be to always use the `promo` field in `events.js`, with either `gems` or `g1g1` so far, and just read from that..?

Reply:
I would've used it - but - the value inside promo is a different one that the banners component name (used in the template) 

if the promo and the name in the component would be the same then we could just use it yes